AdblockPlusSafari/NSString+MarkdownRenderer.h

```diff
@@ -0,0 +1,24 @@
+//
+//  NSString+MarkdownRenderer.h
+//  AdblockPlusSafari
+//
+//  Created by Pavel Zdeněk on 2.O.15.
+//  Copyright © 2015 Eyeo GmbH. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class UIFont;
+
+@interface NSString (MarkdownRenderer)
+
+/**
+@param token a single character representing a span marker. Can be anything but Markdown
+ compatible is `*` for bold and `_` for italic
+@param font the font to substitute in the span
+@return self as attributed string with font changes applied to the found spans
+ */
+- (NSAttributedString*)markdownSpanMarkerChar:(NSString*)markerChar
```

Comment:
We had discussion if to use markdown or html parser some time ago. At the end we decided be to stick with markdown, because parsers seemed simpler to us. We did not take in account share-ability across multiple project.

Honestly I do not give too much attention to proper naming of things, but this class and its name was created by Pavel:]. I think I can rename it.